types/jmuxer/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for jMuxer 2.0
+// Project: https://github.com/samirkumardas/jmuxer
+// Definitions by: Samir Das <https://github.com/samirkumardas>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace JMuxer {
+    interface Options {
+        node: string | HTMLVideoElement;
+        mode?: 'both' | 'audio' | 'video';
+        flushingTime?: number;
+        clearBuffer?: boolean;
+        fps?: number;
+        debug?: boolean;
+        onReady?: () => void;
+    }
+
+    interface Feeder {
+        audio?: Uint8Array;
+        video?: Uint8Array;
+        duration?: number;
+    }
+}
+
+declare class JMuxer {
+    constructor(options: JMuxer.Options);
+    feed(data: JMuxer.Feeder): void;
+    createStream(): any;
```

Comment:
Is there a possibility to type the return value of createStream to `Duplex` or `Writeable` without a referencing node?

I searched some other DefinitelyTyped packages and found that:
```
/// <reference types="node" />
import { Duplex } from "stream";
```

Unfortunately, this approach doesn't work in my project.
I wouldn't spend too much time and keep the any.

Reply:
@Lillifee Thank you for your feedback. I tried Duplex earlier as well but did not work. But the comment `/// <reference types="node" />` did the trick. It is working fine now. You need to review it again for this new change.

Reply:
Really nice, but you accidentally added the Duplex to the ```destroy``` instead of the ```createStream``` function.

Reply:
oops! I have corrected it. Thank you for your patience.